offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -383,7 +383,6 @@ class QuotationDetails {
          * Translates an product item into a HTML row element that can be added to a table
          * @param offerPosition The item position on the offer
          * @param item The product item that is put on the offer
-         * @param affiliationCategory the category of the affiliation
```

Comment:
This should not be removed

Reply:
Strange.. I did not intend to remove it.. added it back